### DISCUSSION TITLE
Code to DGRM using chatgpt

### DISCUSSION THREAD

**Discussion by tonygrud** [AlexeyBoiko/DgrmJS#41] [General]
I am working on an education code tutor using chatgpt.  I can turn code or pysedo code into mermaid syntax accurately and generate flowchart but I have been using your DGRM for making flowcharts on our educational coding site.  My question is do you have any suggestions on the best way forward to generate DGRM flowcharts from code using chatgpt - we have your DGRM code on our website that makes the flowcharts already and I am looking at adding chatgpt with this code to flowchart feature?

**Comment by AlexeyBoiko** [comment]
Hello. Thanks for reaching out.
Please have a look at
https://chatgpt.com/g/g-LSaf72err-dgrm-net
